test/necb/unit_tests/tests/test_necb_skylights.rb

```diff
@@ -139,8 +139,15 @@ def test_necb_skylights()
             # EnergyPlus, OpenStudio & OpenStudio-Standards would report here a
             # SRR% of 4.5%. The effective 'ratio' would vary based on geometry,
             # e.g. larger building footprint, wider overhangs.
-            ratio = gra0.round > graX.round ? skm2 / graX : skm2 / gra0
+            if building == 'SmallOffice'
+              err_msg = "BTAP/OSut: GRA0 <= GRAX (#{cas})?"
+              assert(gra0.round > graX.round, err_msg)
+            else
+              err_msg = "BTAP/OSut: GRA0 != GRAX (#{cas})?"
+              assert(gra0.round == graX.round, err_msg)
+            end
 
+            ratio = skm2 / graX
```

Comment:
Reducing equation - simpler to communicate (see [here](https://github.com/NREL/openstudio-standards/pull/1854#discussion_r1914781609)).